### ISSUE TITLE
Problem display external

### ISSUE THREAD

**Issue by levantanphat9c** [mybigday/react-native-external-display#443]
I'm having a problem that when I bring the app down to the background and reopen it, the external display doesn't show up under the <ExternalDisplay> component anymore, but on the inner screen

**Comment by hansjag04** [comment]
I confirm the issue. Any hint?

**Comment by levantanphat9c** [comment]
in file RNExternalDisplayView.java
  
  @Override
0  public void onHostResume() {
1   // if (displayScreen == null) {
2    //   return;
3   // }
4    updateScreen();
5  }
  
  
 function onHostResume(). I comment line 1,2,3, it is working
 
 i think displayScreen is null so return and don't updateScreen

**Comment by Cyberpaul88** [comment]
It doesn't work.

**Comment by hansjag04** [comment]
Actually, @Cyberpaul88, the solution provided by @levantanphat9c does work. Thanks!
Because this is a hack in the package and I need this in my repo, I decided to copy the folder from the node_modules into a custom folder and link it via NPM/Yarn so if any teammate pulls the repo and installs all libraries, this hack doesn't get overridden by the actual library.

In case you need it, do: 

1. Install the dependency to get all necessary files: `yarn add react-native-external-display`
2. Copy the folder /node_modules/react-native-external-display into your custom folder. Eg., node_custom_modules.
3. Remove the dependency again `yarn remove react-native-external-display`
4. Comment the lines in file `RNExternalDisplayView.java` mentioned in this thread by @levantanphat9c. 
5. Add the custom library to the package.json with `yarn add link:/node_custom_modules/react-native-external-display`
6. Create a custom symlink in the node_modules to your custom path. NPM does this automatically with `npm link react-native-external-display`.
7. Link the library to react native `react-native link react-native-external-display`

Be aware that doing `yarn install` any time will remove any symlink in the node_modules folder (thus, removing the step 6). I decided to solve it with a postinstall script in my package.json that can add the symlink back again after any install:
```json
"scripts": {
   "postinstall": "npm link react-native-external-display"
}
```

Or... comment those lines and submit a PR. However, I don't see this repo very active.. so I better stick with the workaround.

Thanks @levantanphat9c

**Comment by Cyberpaul88** [comment]
Thank you **@[hansjag04](https://github.com/hansjag04)**. Your explanation was clear and I managed to fix my problem. 

Really appreciated

**Comment by jhen0409** [comment]
Confirmed and it caused by 9a3413f35c872773beb8f2768da59868e934a9c8.

Fixed in 8a01beb927bfd5d86be563a627fae4ae3567eb68. Thanks for report!
